thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/anomaly/task/TaskConstants.java

```diff
@@ -26,15 +26,8 @@
     DETECTION,
     DETECTION_ALERT,
     YAML_DETECTION_ONBOARD,
-    ANOMALY_DETECTION,
-    MERGE,
     // TODO: deprecate ALERT task type
-    ALERT,
-    ALERT2,
-    MONITOR,
-    DATA_COMPLETENESS,
-    CLASSIFICATION,
-    REPLAY
+    MONITOR
```

Comment:
Add a simple description for each task type.

Reply:
Added one line description each type.